AppCenterDistribute/AppCenterDistribute/Support/AppCenterDistribute.xcconfig

```diff
@@ -1,6 +1,6 @@
 PRODUCT_BUNDLE_IDENTIFIER = com.microsoft.appcenter.distribute
 
-OTHER_LDFLAGS=$(inherited) -framework Foundation -weak_framework SafariServices
+OTHER_LDFLAGS=$(inherited) -framework Foundation -framework AuthenticationServices -weak_framework SafariServices
```

Comment:
I guess it should be `weak_framework` because it's not available on old iOS versions